### DISCUSSION TITLE
Database access with Datagrip

### DISCUSSION THREAD

**Discussion by Scott-Gardner** [gravitational/teleport#11141] [General]
Hi All

I have a working Teleport cluster.  The only issue I am having is when I try to connect Datagrip to a MySQL 8 database.  I get the error:
    [HY000][1193] Unknown system variable 'query_cache_size'.
I have tried a number of fixes I found wile researching but nothing has helped so far.  I am able to connect Workbench, SQLAce and other IDEs but not Datagrip.

Has anyone come across this before or have any thoughts on a potential fix?

Thanks

**Comment by scottpgallagher** [comment]
I think that query_cache_size was something that was removed in mysql 8.0.3; so older version 8.0.2 and older will work; There currently is an open issue for this if you would like to chime in on that issue to get more visibility to our engineering team.

**Reply by scottpgallagher** [reply]
https://github.com/gravitational/teleport/issues/6584

**Reply by Scott-Gardner** [reply]
Awesome thanks I'll take a look
